data/messages.proto

```diff
@@ -136,7 +136,8 @@ message SendWalletCoinsRequest {
 
 message PayInvoiceRequest {
     int64 amount = 1;
-    string paymentRequest = 2;
+    int64 fee = 2;
```

Comment:
In general it is suggested not to change existing numbers of fields but add as additional numbers to make it backward compatible.